okhttp/src/main/java/okhttp3/internal/cache/CacheStrategy.java

```diff
@@ -160,7 +160,7 @@ public Factory(long nowMillis, Request request, Response cacheResponse) {
           } else if ("ETag".equalsIgnoreCase(fieldName)) {
             etag = value;
           } else if ("Age".equalsIgnoreCase(fieldName)) {
-            ageSeconds = HttpHeaders.parseSeconds(value, -1);
+            ageSeconds = UtilKtKt.toNonNegativeInt(value, -1);
```

Comment:
`UtilKtKt` isn't a great name.

Reply:
Give that one a minute; it’s callers will soon all be Kotlin.

Reply:
Fixed to be only one Kt.